sorc/ncep_post.fd/PARA_RANGE.f

```diff
@@ -1,14 +1,5 @@
 !> @file
-!> @brief para_range() sets up decomposition values.
-!>
-!> This subroutine sets up decomposition values.
-!>
-!> @param[in] N1 First interate value.
-!> @param[in] N2 Last interate value.
-!> @param[in] NPROCS Number of MPI tasks.
-!> @param[in] IRANK My taks ID.
-!> @param[out] ISTA First loop value.
-!> @param[out] IEND Last loop value.
```

Comment:
Note that this info is a duplicate of information included directly above the para_range() subroutine.